packages/jaspr/lib/dom.dart

```diff
@@ -0,0 +1,7 @@
+export 'jaspr.dart' hide runApp, Document;
```

Comment:
I'd highly recommend dropping this export. It will cause inconsistencies in how people access members from `jaspr.dart`.

It'll also result in dom.dart showing up as a duplicate option when adding imports for items from `jaspr.dart`, requiring developers to have to keep making extra choices.